extensions/amp-lightbox-gallery/0.1/test/integration/test-amp-lightbox-gallery.js

```diff
@@ -14,6 +14,7 @@
  * limitations under the License.
```

Comment:
You'll get a merge conflict from https://github.com/ampproject/amphtml/pull/16824 here because this test suite is super flaky on Windows. Sadness, but super thanks for writing tests! I'll take a look and see what we can do to fix the Windows thing in a bit.